jenkins/indy-email-verification/Jenkinsfile.pipeline.param

```diff
@@ -1,11 +1,12 @@
 #=========================================================
 # OpenShift Jenkins pipeline template parameters for:
-# Jenkinsfile: ./jenkins/indy-email-verification/Jenkinsfile
+# Jenkinsfile: ../jenkins/indy-email-verification/Jenkinsfile
 # Template File: https://raw.githubusercontent.com/BCDevOps/openshift-tools/master/provisioning/pipeline/resources/pipeline-build.json
 #=========================================================
 NAME=indy-email-verification
 # GITHUB_WEBHOOK_SECRET=[a-zA-Z0-9]{40}
-SOURCE_REPOSITORY_URL=https://github.com/bcgov/indy-email-verification.git
+# GENERIC_WEBHOOK_SECRET=[a-zA-Z0-9]{40}
+SOURCE_REPOSITORY_URL=https://github.com/wadeking98/indy-email-verification.git
```

Comment:
Check URls for the GIT repositories, they should point to `bcgov`

Reply:
changed pipeline to point to bc gov